services/mailgun.js

```diff
@@ -0,0 +1,42 @@
+const Mailgun = require('mailgun-js');
+require('dotenv').config()
+
+
+//Your api key, from Mailgun’s Control Panel
+const api_key = process.env.API_KEY
+
+//Your domain, from the Mailgun Control Panel
+
+const domain = process.env.DOMAIN
+
+//Your sending email address
+const from_who = process.env.FROM_WHO;
```

Comment:
good variable names should make adding a comment unnecesssary. If you used `SENDER_EMAIL`, then you don't need the comment.